product_import/__manifest__.py

```diff
@@ -13,9 +13,12 @@
         "stock",
         # OCA/edi
         "base_business_document_import",
+        # OCA/queue
+        "queue_job",
```

Comment:
I've thought again about this... We should remove this dependency IMO.
In the simplest case we won't need it, in the more complex case we'll probably use the framework. Then, we can leverage the framework to the split. 
Additionally, there's now a new feature in queue_job which is called `split` and allows to.... split items in batches.
See https://github.com/OCA/queue/pull/658